2-dom-manipulation/form.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="es">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css">
+    <title>Html Form</title>
+  </head>
+  <body class="main-form">
+    <header>
+      <h1>Simple Form</h1>
+    </header>
+    <main>
+      <form class="form">
+        <label class="label" for="name"> 
+          First Name <br />
+          <input type="text" name="name" id="name" required />
+        </label>
+        <label class="label" for="lastName">
+          Last Name <br />
+          <input type="text" name="lastName" id="lastName" required />
+        </label>
+        <label class="label" for="email">
+          Email <br />
+          <input type="email" name="email" id="email" required />
+        </label>
+        <label class="label" for="dateOfBirth">
+          Date of Birth <br />
+          <input type="date" name="date" id="dateOfBirth" required />
```

Comment:
You can avoid `<br>` with CSS (flex column) or using divs within the `<label>` tag